.github/workflows/test-pr-command.yml

```
@@ -0,0 +1,163 @@
+  AIRBYTE_ANALYTICS_ID: ${{ vars.AIRBYTE_ANALYTICS_ID }}
+
+jobs:
+  start-workflow:
+    name: Append 'Starting' Comment
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get PR JSON
+      id: pr-info
+      env:
+        GH_TOKEN: ${{ github.token }}
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2002:style:3:17: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead

(shellcheck)